benchmarks/msg_pingpong.c

```diff
@@ -193,7 +193,7 @@ static int run(void)
 			goto out;
 	}
 
-	ft_finalize();
+	ret = ft_finalize();
 out:
 	fi_shutdown(ep, 0);
```

Comment:
What's the best way to handle failure of `fi_shutdown`? Should the shutdown still happen if the finalize failed?